packages/core/test/scxml.test.ts

```diff
@@ -168,30 +168,30 @@ const testGroups = {
     // 'test210.txml', // sendidexpr not supported yet
     // 'test215.txml', // <invoke typeexpr="...">
     // 'test216.txml', // <invoke srcexpr="...">
-    // 'test220.txml', // done.invoke used as inexact event descriptor
+    'test220.txml',
     // 'test223.txml', // idlocation not implemented yet
-    // 'test224.txml', // done.invoke used as inexact event descriptor
+    // 'test224.txml', // <invoke idlocation="...">
     // 'test225.txml', // unique invokeids generated at invoke time
     // 'test226.txml', // <invoke src="...">
-    // 'test228.txml', // done.invoke used as inexact event descriptor + _event.invokeid not implemented yet
+    'test228.txml',
     'test229.txml',
-    // 'test230.txml', // done.invoke used as inexact event descriptor + SCXML _event properties not implemented yet
-    // 'test232.txml', // done.invoke used as inexact event descriptor
+    'test230.txml',
+    // 'test232.txml', // TODO: not sure
```

Comment:
This test is passing now and I don't see anything much special about it so let's assume that it has not been passing for you in the middle of development due to some minor bug
```suggestion
    'test232.txml',
```